Services/IConsentService.cs

```diff
@@ -0,0 +1,57 @@
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Http.Features;
+using OrchardCore.Users;
+using System.Security.Claims;
+using System.Threading.Tasks;
+
+namespace Lombiq.Privacy.Services
+{
+    /// <summary>
+    /// Service to manage user privacy consent.
+    /// </summary>
+    public interface IConsentService
+    {
+        /// <summary>
+        /// An asynchronous operation that decides whether the consent banner should be displayed based on
```

Comment:
Kb. 120 karakternél legyen törve az összes comment. A ReWrap VS extension ezt magától megcsinálja.

Reply:
Nem kell redundáns magyarázat a metódushoz, szal elég csak "Decides whether the consent banner should..."